### ISSUE TITLE
[REQUEST] Wireshark, OpenXenManager, Hangouts, Terminix

### ISSUE THREAD

**Issue by kunamashina** [keeferrourke/la-capitaine-icon-theme#57]
Very good job on my first request :+1: 

Looking forward for your new additions

**Comment by keeferrourke** [comment]
can you paste the contents of the openXenManager *.desktop file from `/usr/share/applications/`?

Hangouts and Terminix already have icons, but I can do redesigns similar to the request from #56

**Comment by kunamashina** [comment]
- openxenmanager.desktop
- wireshark-gtk.desktop
- terminix has the same icon with your icon theme as this URL: https://plus.google.com/+WorldofGnomeOrg/posts/B2jKqyN3HtD
- hangouts look the same as default one for me, it is smaller than the rest of the icons see in the URL:
http://imgur.com/a7GWYgK

Looking forward to your commitment, keep up the good work

**Comment by keeferrourke** [comment]
@kunamashina can I get the _contents_ of the the files you listed there? specifically the `Icon=...` lines

regarding hangouts, I have the icons for a lot of Google Chrome apps in La Capitaine, I just need to symlink them to the right names - this should be done tonight \*fingers crossed\*

**Comment by kunamashina** [comment]
# **openxenmanager**
[Desktop Entry]
Categories=RemoteAccess;Monitor;
Encoding=UTF-8
Name=OpenXenManager
GenericName=manage Citrix XenServer hosts
Type=Application
Exec=/usr/bin/openxenmanager
Icon=openxenmanager

# **Wireshark**
[Desktop Entry]
X-SuSE-translate=true
# The format of this file is specified at
# http://standards.freedesktop.org/desktop-entry-spec/1.0/
# The entries are in the order they are listed in version 1.0
Type=Application
# This is the version of the spec for this file, not the application version.
Version=1.0
Name=Wireshark (GTK)
GenericName=Network Analyzer
Comment=Network traffic analyzer
Icon=wireshark-gtk
TryExec=wireshark-gtk
Exec=/usr/bin/xdg-su -c /usr/bin/wireshark-gtk %f
Terminal=false
MimeType=application/vnd.tcpdump.pcap;application/x-pcapng;application/x-snoop;application/x-iptrace;application/x-lanalyzer;application/x-nettl;application/x-radcom;application/x-etherpeek;application/x-visualnetworks;application/x-netinstobserver;application/x-5view;application/x-tektronix-rf5;application/x-micropross-mplog;application/x-apple-packetlogger;application/x-endace-erf;application/ipfix;application/x-ixia-vwr;
# Category entry according to:
# http://standards.freedesktop.org/menu-spec/1.0/
Categories=Network;Monitor;Qt;

Can't find terminix where is the application content installed at ... >.<

Someone has an idea?

**Comment by Elbullazul** [comment]
@kunamashina you can search in the icons theme folder in apps. Maybe it's there.  Or in the hi color icon theme. Anyways you can use the following command to locate all terminix related files.  
```shell
Locate *terminix*
```

**Comment by keeferrourke** [comment]
@kunamashina I figured out where terminix was found and updated the icon name in commit 09c43ab8643ba9f039dcd155cb8bf710ff06823c, it seems that at least on Ubuntu the desktop entry looks for  `com.gexperts.Terminix.svg`.

Also check out commit 5698cd034e300bd1da35cb60e88260de735a20dd for a bunch of updated Chrome/Chromium extension icons!

_Note that these icons will only theme apps that were added by the first Chromium user_ - if your Chromium browser has multiple user profiles and they add shortcuts to the desktop, those shortcuts will not be themed. This is because Chromium insanely installs copies of the desktop entry which look for `Icon=<filename>-Profile#`, where `#` is the number of profiles succeeding the `Default` profile. I cannot reasonably support this if someone's Chromium has a ridiculous number of users with apps installed.

TL;DR Chromium apps are now themed as long as Chromium only has one user profile.

**Comment by kunamashina** [comment]
terminix and Terminix ... blah :+1: that's the reason I couldn't find it

Icon=com.gexperts.Terminix

Keeferrourke found it in the end hehe

PS: 
Is there an easy way to update the icons? Or should I wait for the rest of the icons wireshark and openxenmanager?

**Comment by keeferrourke** [comment]
@kunamashina if you cloned from git, just do
```
cd ~/.icons/la-capitaine-icon-theme
git pull
```
and then set the theme again using your preferred desktop tools. This is sort of desktop independent, but you can use `gnome-tweak-tool` or `lxappearance`.

If you are using GNOME/Unity/some other GNOME-like DE, restarting the desktop shell should refresh the icons for you. You can do this by pressing `ALT-F2` to open the run dialogue, and issuing a command like `r` or `restart`.

**Comment by keeferrourke** [comment]
Wireshark and OpenXenManager are on their way; I want to do something creative with both of them though, so it might be a few days before they are themed.

**Comment by keeferrourke** [comment]
@kunamashina check out commit https://github.com/keeferrourke/la-capitaine-icon-theme/commit/30dcb6b5bffe73e31cdab3299b20678efc7d75f6 for the new wireshark icon. I based this off of the [emoji one shark](http://emojione.com/wp-content/uploads/assets/emojis/1f988.svg), but simplified the design, added subtle gradients and shadows, and made it a bit cuter.

![wireshark svg](https://cloud.githubusercontent.com/assets/6888381/21078050/d76885dc-bf2e-11e6-80f8-cea0e6d22043.png)
`wireshark preview`

I thought this was a creative, fun solution, but if you want an alternative icon that more closely resembles the original logo, let me know and I can make one much more quickly.

Note: I'll be writing exams until the end of the month, so bear with me and eventually I'll get OpenXenManager done :)

**Comment by kunamashina** [comment]
Hi, 

That is fantastic ! Great work, very much appreciated. Although I would prefer a more serious icon since this is the software that pros only use. And therefore a serious icon like in the link below would be the one I opt for

https://goo.gl/images/yuB1B3

Looking forward to hearing from you 

Thanks in advance and good luck with your exams

**Comment by keeferrourke** [comment]
@kunamashina I like to think that the pros have a sense of humour, but I'll get a more serious alternative icon up. If it's not up tomorrow, just send me an email reminder :)

**Comment by kunamashina** [comment]
Anything new in the meantime bro?

**Comment by keeferrourke** [comment]
Not as of late... I've been busy. I can't seem to find an icon reference for OpenXenManager however, and it (at least the Ubuntu package) doesn't come with a desktop file... so I'm unable to theme it at this time.

I can do an alternate design for wireshark as requested, but I'm not sure how to approach this. I'd like to keep the cute icon as the default &ndash; would pushing a `wireshark-alt.svg` to the repo be sufficient? You would need to rename some files yourself to apply the "professional" icon you requested.

**Comment by kunamashina** [comment]
That's ok :) 👍

**Comment by keeferrourke** [comment]
@kunamashina there's the alternate icon, finally :)
You just need to rename the files as you please to apply the correct one - pulling to update La Capitaine shouldn't cause any conflicts with your local copy of the theme, as I don't think I'll be modifying the wireshark icons anytime in the foreseen future.
